### ISSUE TITLE
com.google.fonts/check/monospace in name.py?

### ISSUE THREAD

**Issue by guidoferreyra** [googlefonts/fontbakery#3606]
The monospace check is on name table profile should not be in something like Universal Default section? 

https://github.com/googlefonts/fontbakery/blob/1a6848982faa480933099adda60a0075e78afcc0/Lib/fontbakery/profiles/name.py#L108

**Comment by felipesanches** [comment]
yeah... this check initially was placed there because it belonged to the opentype profile (things that required by the Open Type specification). And, in particular, in the `name.py` file because the opentype profile is organized that way, by primary table that a check validates.

But this check actually deals with requirements for many different tables so it is a bit hard to define where it should be placed within the opentype profile.

**Comment by guidoferreyra** [comment]
Thanks for the explanation :)
